File: 241008/점수 입출력/enter-int-and-print-score.js
const fs = require("fs");

let score = Number(fs.readFileSync(0).toString().trim());

if(score >= 1 && score <= 1000){
    console.log(`Your score is ${score} point.`)
}
else{
    console.log(`입력값이 유효한 범위를 벗어났습니다.`)
}

File: 241010/점수 입출력/enter-int-and-print-score.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        if(n >= 1 && n <= 1000){
            System.out.printf("Your score is %d point\n", n);
        }
    }
}